### PR TITLE
feat(explorer-shared): Moved entities and migrations to shared project

### PR DESCRIPTION
**Pull request by Malak67**
This solution moves all entities and migrations into the shared package.
Hence, when starting the indexer app, this needs to be started from the root via this command:
`yarn indexer-start`
This will first run any migration found inside the shared package and then starts the indexer app.

### CODE REVIEW

**Comment by VargaElod23** [comment]
With this, when creating the according Dockerfiles for any package that uses the shared one, the building scope needs to be 
 `taraxa-platform root`. Just a note

File: packages/explorer-shared/tsconfig.json [hunk 1]
```diff
@@ -16,7 +20,11 @@
     "module": "commonjs",
     "resolveJsonModule": true,
     "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true
+    "strictPropertyInitialization": false,
+    "forceConsistentCasingInFileNames": true,
```
Comment (VargaElod23) [inline packages/explorer-shared/tsconfig.json:24]: Hmm why was this necessary to define?

**Review by VargaElod23 — APPROVED** [review]
We should do an NPM release for this as adding this change makes management a bit more complex. To avoid having all of this into consideration when building packages that depend on it, it would be great if the package could be directly pulled from yarn or npm.

File: packages/explorer-shared/tsconfig.json (hunk 1 above)
Comment (Malak67) [reply packages/explorer-shared/tsconfig.json:24]: I think it was there initially anway. It usually added by default from NestJs when creating a new project so I thought this project should have a similar tsconfig as the projects using NestJs.

Comment (VargaElod23) [reply packages/explorer-shared/tsconfig.json:24]: OKay. I'm fine with that
